packages/terra-dialog/tests/jest/Dialog.test.jsx

```diff
@@ -11,9 +11,9 @@ describe('Dialog', () => {
   });
 
   // Prop Tests
-  it('should use the default value when no value is given', () => {
+  it('should render with the provided props', () => {
```

Comment:
These first two tests are identical.

Reply:
Updated some of these: https://github.com/cerner/terra-core/pull/1173/commits/a74b4cb2f1c6bf7a2fd274272909de8b940bf0ca